libs/wire-api/src/Wire/API/Event/Team.hs

```diff
@@ -232,15 +241,19 @@ parseEventData TeamCreate Nothing = fail "missing event data for type 'team.crea
 parseEventData TeamCreate (Just j) = EdTeamCreate <$> parseJSON j
 parseEventData TeamUpdate Nothing = fail "missing event data for type 'team.update'"
 parseEventData TeamUpdate (Just j) = EdTeamUpdate <$> parseJSON j
-parseEventData CollaboratorAdd Nothing = fail "missing event data for type 'team.collaborator-add"
+parseEventData CollaboratorAdd Nothing = fail "missing event data for type 'team.collaborator-add'"
 parseEventData CollaboratorAdd (Just j) = do
   let f o = EdCollaboratorAdd <$> o .: "user" <*> o .: "permissions"
   withObject "collaborator add data" f j
 parseEventData AppCreate Nothing = fail "missing event data for type 'team.app-create'"
 parseEventData AppCreate (Just j) = do
   let f o = EdAppCreate <$> o .: "user"
   withObject "app create data" f j
-parseEventData CollaboratorRemove Nothing = fail "missing event data for type 'team.collaborator-remove"
```

Comment:
this looks like you've dropped the `CollaboratorRemove Nothing` case?

Reply:
oops, merge issues, thanks.